### PR TITLE
Cgm 1312 versioning

### PR DESCRIPTION
**Pull request by fooni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,24 @@ Please go through wiki pages for [detailed setup of local development](https://g
 | feature | The master branch tracks released code only. The only commits to master are merges from release branches and hotfix branches.                                                                                                                                                                       |
 
 For more detailed explaination, please go through [branch policy document](https://github.com/Welthungerhilfe/cgm-rg/wiki/Branch-Policy)
+
+## Versioning
+This project is versioned according to CGM [versioning rules](https://dev.azure.com/cgmorg/ChildGrowthMonitor/_wiki/wikis/ChildGrowthMonitor.wiki/185/Versioning-Release-management). The version of the project
+can be found in `VERSION`. The CI pipeline tags and pushes a patch version update automatically
+after a merge to the `main` branch.
```
Comment (gangooteli) [inline README.md:32]: cgm-rg doesn't have a main branch.

File: ci/azure-pipelines.yml [hunk 2]
```diff
@@ -66,13 +87,43 @@ stages:
               testResultsFormat: 'JUnit'
               testResultsFiles: '**/TEST-*.xml'
               failTaskOnFailedTests: true
+
+  - stage: Version
+    displayName: Tag & bump version
+    dependsOn: Verify
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
```
Comment (gangooteli) [inline ci/azure-pipelines.yml:94]: cgm-rg does not have a main branch

File: ci/azure-pipelines.yml [hunk 3]
```diff
@@ -66,13 +87,43 @@ stages:
...
+              # Setup Git
+              git config --global user.email "noreply@childgrowthmonitor.org"
+              git config --global user.name "CGM CI"
+              # Tag current commit
+              CURR_VERSION=$(head -n 1 VERSION)
+              git tag -a "v$CURR_VERSION" -m "Result Generation version $CURR_VERSION"
+              # Bump PATCH version
+              bumpversion --commit patch
+              # Push tag and bumped version
+              git push --follow-tags origin HEAD:main
```
Comment (gangooteli) [inline ci/azure-pipelines.yml:116]: cgm-rg doesn't have a main branch

**Review by gangooteli — CHANGES_REQUESTED** [review]
cgm-rg does not have main branch. Should we use develop branch instead of main?

**Review by gangooteli — APPROVED** [review]
LGTM. 
Also renamed develop branch to main branch

**Review by shanky0009 — APPROVED** [review]

**Pull request merged**